smashing-magazine/smash-out.csv

```diff
@@ -0,0 +1,518 @@
+title,author,tags,link,source
```

Comment:
This big CSV file is created by your script below, it appears. Does it really need to be checked into source? Seems like it can be safely `gitignored`. @masungwon @kenzable